### PR TITLE
Add requirement for review costs management

### LINKED ISSUE #494: The "what you need to do" sections of "Welcome contributors" and "Require review of contributions" lack guidance regarding budgeting for outside contributions
Codebases start with a small set of contributors, possibly a single small development team.

Without guidance regarding reasonable expectations of budget for possible outside contribution, the Standard for Public Code could be interpreted as creating a larger demand upon the initial developers than is likely intended. Also, without any time budgeted for review of new participants' contributions, these new developers will not be able to genuinely contribute.

Some guidance in this area could help make implementing the Standard itself more easy to reason about and (more importantly) help to ensure that initial procurement is sufficient.

### PR DESCRIPTION
**Pull request by Ainali**
Fixes #494

### CODE REVIEW

**Comment by ericherman** [comment]
I think this has the right ideas and I also think that the wording could be improved.

File: criteria/welcome-contributors.md [hunk 1]
```diff
@@ -27,6 +28,7 @@ Newcomers need to understand and trust the codebase community’s governance.
 * Confirm that it is possible to submit suggestions for changes to the codebase.
 * Confirm there are contribution guidelines.
 * Confirm that the codebase governance is clearly explained, including how to influence codebase governance.
+* Check that contributing guidelines covers who is expected to cover the costs of reviewing contributions.
```
Comment (ElenaFdR) [inline criteria/welcome-contributors.md:31]:
```suggestion
* Check that contributing guidelines cover who is expected to cover the costs of reviewing contributions.
```

File: criteria/welcome-contributors.md [hunk 2]
```diff
@@ -41,13 +43,15 @@ Newcomers need to understand and trust the codebase community’s governance.
 
 * Make sure that the documentation of the governance includes the current process for how to make changes to the governance.
 * If the community has some consensus about how the governance should change, then include those ideas stated as ambitions in the documentation.
+* Describe how the codebase community wants to handle the review of incoming contributions. If needed, secure and allocate budget for that process.
```
Comment (ElenaFdR) [inline criteria/welcome-contributors.md:46]:
```suggestion
* If needed, make sure you have allocated budget for the contributions review process as agreed by the codebase community.
```

**Review by ElenaFdR — COMMENTED** [review]
Interesting and useful addition!

**Review by ElenaFdR — APPROVED** [review]

**Review by ericherman — APPROVED** [review]

**Pull request merged**